documentation/prerequisites.md

```diff
@@ -4,24 +4,40 @@ title: Prerequisites
 
 Following is the list of prerequisites for running Kubernetes with vSphere Cloud Provider:
 
-* Kubernetes version - 1.6.5 onwards. Tested with v1.6.5, v1.7, v1.8. 
-* vSphere version - 6.0.x onwards.
-* Container Host operating systems - 
-    - Photon - v1.0 GA 
-    - Ubuntu - v16.04 
-    - CoreOs - v4.11.2 
+* We recommend Kubernetes version 1.6.5 and above
+
+Following table summarizes major feature introduction in Kubernetes releases. This can help identify Kubernetes version for specific vSphere Cloud Provider feature.
+
+| Kubernetes Release | vSphere Cloud Provider feature |
+| ------ | ------ |
+| v1.6.3 | Dynamic volume provisioning using vSAN storage capabilities |
+| v1.6.5 | vSphere HA support |
+| v1.7.0 | vSphere storage policy support for dynamic volume provisioning |
+| v1.7.0 | Metric collections for vsphere Storage API latency and errors |
+| v1.8.0 | Major vSphere Cloud Provider refactoring for better debuggability, logging     and code maintenance |
+| v1.8.2 | Major performance improvement for large scale deployment |
+| v1.9.0 | Multi vCenter Support |
+
+* vSphere version 6.0.x and above
+
+* Container Host operating systems -
+    - Photon - v1.0 GA
+    - Ubuntu - v16.04
+    - CoreOs - v4.11.2
     - Centos  - v7.3
     - RHEL OS -  v7.4
+
 * vSphere setup to deploy the Kubernetes cluster.
-* vSphere shared storage. It can be any one of VSAN, sharedVMFS, sharedNFS.
-   - Shared storage allows all the nodes in the Kubernetes cluster to access the storage blocks.
+   - For kubernetes version 1.9.x and above: vSphere Cloud Provider supports kubernetes cluster spanning across multiple vCenters.
+   - For kubernetes version 1.8.x and below: vSphere Cloud Provider supports kubernetes cluster deployed only in one vCenter.
+* vSphere shared storage.
```

Comment:
Please reword to "vSphere supported storage"

Reply:
Done